rusage.c

```diff
@@ -38,13 +38,20 @@ static double get_aggregate_proc_stat_system_time()
         unsigned long dummy, system, irq, softirq;
         int nscan = 0;
         FILE *f = fopen("/proc/stat", "r");
-        if (f) {
-              /* Just keep the important values of the first seven */
-              nscan = fscanf(f, "cpu %lu %lu %lu %lu %lu %lu %lu",
```

Comment:
You're right. Apologies for the misunderstanding. I'll close the PR.

Since I was mistaken, would it also be appropriate to delete issue #16? Thank you for your understanding.